src/connectors/tests/http/client.rs

```diff
@@ -553,9 +601,23 @@ async fn http_client_request_auth_basic() -> Result<()> {
     Ok(())
 }
 
-#[async_std::test]
+/*
+
+What we know:
+
+1) we send data
+2) the data is received by the server
+3) the server responds with the data it received
+4) the data is received by the client
+5) the client sends the response to the pipeline
+
+6) the pipele reads the data, but hangs when trying to read it again (for some reason, this is where it gets confusing)
```

Comment:
`the pipeline`

Reply:
whacking this as it was just notes from a debugging session, good catch!